bolt-google-cloud-functions/pom.xml

```diff
@@ -0,0 +1,54 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This pom.xml is based on https://cloud.google.com/functions/docs/first-java This Maven project is not part of the multi-module maven project. It works independently (this means it downloads jars from the Maven Central and boots the app without any extra settings).